assets/quicknavi.js

```diff
@@ -2,6 +2,15 @@ $(function () {
     quicknavi_filter_init();
     $(document).on('pjax:end', function() {
        quicknavi_filter_init();
+       var ctype = getUrlVars()["ctype"];
+       if (ctype)
+       {
+	   $(function() {
```

Comment:
Der Filter funktioniert nicht, wenn in einem Addon pjax abgeschaltet ist. Gibt es eine andere Möglichkeit? @staabm